libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -129,6 +129,7 @@ void AC_Avoid::adjust_velocity_fence(float kP, float accel_cmss, Vector3f &desir
     
     // limit acceleration
     const float accel_cmss_limited = MIN(accel_cmss, AC_AVOID_ACCEL_CMSS_MAX);
+    (void)accel_cmss_limited;  // in case both Beacon and Fence are compiled out
```

Comment:
Txs for this PR.

I look at this line and I'm not sure what's its doing actually.. maybe we should just surround it with something like this?

#if AP_FENCE_ENABLED || AP_BEACON_ENABLED
#endif

Reply:
> I look at this line and I'm not sure what's its doing actually.. maybe we should just surround it with something like this?

Hmmm.  This is basically a very short way of "using" the variable.

It's not an unusual pattern in our code - there are 48 other instances.

> #if AP_FENCE_ENABLED || AP_BEACON_ENABLED #endif

This will work - at the expense of adding several lines, and adding the overhead of maintaining the #if based on the code below.  It does have the benefit of retaining the static checks in the case that you do have fence or beacon in.

Did you still want me to change this?